Sources/PayPal/PayPalClient.swift

```diff
@@ -37,31 +39,37 @@ public class PayPalClient {
     ///   - request: the PayPalRequest for the transaction
     ///   - presentingViewController: the ViewController to present PayPalPaysheet on, if not provided, the Paysheet will be presented on your top-most ViewController
     ///   - completion: Completion block to handle buyer's approval, cancellation, and error.
-    public func start(
-        request: PayPalRequest,
-        presentingViewController: UIViewController? = nil,
-        completion: @escaping (PayPalCheckoutResult) -> Void
-    ) {
+    public func start(request: PayPalRequest, presentingViewController: UIViewController? = nil) {
         CheckoutFlow.set(config: config, returnURL: returnURL)
 
         CheckoutFlow.start(
```

Comment:
@sarahkoop I would do it the other way around, because the Web Flow PR is blocked from merging until further notice. So i would merge this, and then "update" the web flow.